tutorials/katacoda/1-serving-intro/step1.md

```diff
@@ -0,0 +1,20 @@
+## Installation
+> The startup script running on the right will install the `kn` cli and wait for kubernetes to start. Once you see a prompt, you can click on the commands below at your own pace, and they will be copied and run for you in the terminal on the right.
+
+1. Install Knative Serving's core components
+    ```
+    kubectl apply --filename https://github.com/knative/serving/releases/download/v0.19.0/serving-crds.yaml
```

Comment:
harcoding the version would be hard to maintain `v0.19.0` is located in 23 places in this PR

@abrennan89 Do you know if there is a variable we can use per branch to extract the `0.19` ?

Reply:
@markusthoemmes is probably a better person to ask

Reply:
elsewhere in the docs, the version number is referred like this: `kubectl apply --filename {{< artifact repo="serving" file="serving-crds.yaml" >}}`. That works great in the tooling used by the website, however,  these files are going to be consumed by katacoda, hence it is not possible to use that same syntax. 

How do we feel about updating these versions as part of the release process?

Reply:
Yeah that a possibility to update for each release cut, but would need to be automated with the scripts that cut a release or update api ref spec @RichieEscarez migjt have guidance on location of scripts

Reply:
I still see issues having this pointing to an old version of knative. For example we cut version 0.21

Perhaps what do you think if we don't hardcore a specific version and instead we use the nightly URL instead?

Reply:
Oh wait this is running on Linux not users computer, I proposed you use bash/sh for example cusinf curl to find out the latest cut version and stored that in variable then use the variable when installing with kubectl URL